modules/wsl-distro.nix

```
@@ -9,6 +9,7 @@ in
 
   options.wsl = with types; {
     enable = mkEnableOption "support for running NixOS as a WSL distribution";
+    useWslLib = mkEnableOption "/usr/lib/wsl/lib as OpenGL driver";
```

```suggestion
    useWindowsDriver = mkEnableOption "/usr/lib/wsl/lib as OpenGL driver";
```
or maybe something with Graphics? Not sure but he current name is no that descriptive 

```suggestion
    useWslLib = mkEnableOption "Use windows host as OpenGL driver";
```